src/marqo/tensor_search/tensor_search.py

```diff
@@ -1703,7 +1723,7 @@ def create_bulk_search_response(queries: List[BulkSearchQueryEntity], query_to_b
     return results
 
 def _vector_text_search(
-        config: Config, index_name: str, query: Union[str, dict], result_count: int = 5, offset: int = 0,
+        config: Config, index_name: str, query: Optional[Union[str, dict]] = None, result_count: int = 5, offset: int = 0,
```

Comment:
why is query set to None by default here? I think we should be really careful about adding default params

Reply:
you're right, `search` should already make query `None` if it doesn't exist, then pass that `None` to `_vector_text_search`. Fixed.